pkg/highlight/parser.go

```diff
@@ -298,6 +298,36 @@ func ParseDefFromFile(f config.RuntimeFile, header *Header) *Def {
 	return syndef
 }
 
+// FindRealRuntimeFile can be used to find a specific definition in the user's
+// custom syntax files
+func FindRealRuntimeFile(name string, header *Header) *Def {
```

Comment:
These return a Def, not a RuntimeFile. So, rather something like `FindDefInRealRuntimeFiles()` and `FindDefInRuntimeFiles()`?

OTOH, that's a mouthful... If we move these functions to `internal/buffer/buffer.go` before `UpdateRules()`, as I've suggested, then maybe let them be named `findRuntimeFile()` and `findRealRuntimeFile()` even though they return syntax def, not a file (since it will be clear from the context what are they for)... Or maybe even such way then: let `findRuntimeFile()` and `findRealRuntimeFile()` return both the file itself and the syntax def parsed from it, even though we will ignore the returned file value.

Reply:
Then me make it more explicit what the function shall reflect: `find*SyntaxDef()`

Reply:
Yeah, sounds good.